doc/policies.md

```diff
@@ -53,6 +53,19 @@ Notice that we did not indicate what APIcast does in the `init` and the
 request. `init` is executed when APIcast boots, and `init_worker` when each
 of each of its workers start.
 
+The order in the policy chain affects the behavior of certain policies.
```

Comment:
Maybe expand it like this:
The order in which policies actions are applied depend on two factors:
- position of the policy within the policy chain;
- the phase in which the policies act.
This means that sometimes the outcome of the policies execution may be affected by other policies which are located further in the policy chain.

Reply:
Good idea :+1: 

I've added it.